src/parser/PropertyPath.h

```diff
@@ -106,6 +123,8 @@ class PropertyPath {
   bool isIri() const;
 
   Operation operation_;
+  int min_;
+  int max_;
```

Comment:
Probably we should use an unsigend type (like `size_t`) here, and do the checking for positive values etc. in the Visitor.